src/lib/alarmMgr/AlarmManager.cpp

```diff
@@ -101,6 +101,29 @@ int AlarmManager::semInit(void)
 
 /* ****************************************************************************
 *
+* AlarmManager::semGet - 
+*/
+const char* AlarmManager::semGet(void)
+{
+  int value;
+
+  if (sem_getvalue(&sem, &value) == -1)
+  {
+    return "error - state not available";
```

Comment:
If this at the end could be printed in the JSON, I'd leave just "error". Maybe I'd rewrite as:

```
if (sem_getvalue(&sem, &value) == -1)
{
  // State not available
  return "error";
}
```